client/build_prune.go

```diff
@@ -22,9 +22,6 @@ func (cli *Client) BuildCachePrune(ctx context.Context, opts build.CachePruneOpt
 		query.Set("all", "1")
 	}
 
-	if opts.KeepStorage != 0 {
-		query.Set("keep-storage", strconv.Itoa(int(opts.KeepStorage)))
-	}
```

Comment:
This probably won't work well, as it would remove the feature for older API versions (which the client supports)